examples/tutorials/contour-map.py

```diff
@@ -0,0 +1,73 @@
+"""
+Creating a map with contour lines
+=================================
+
+Plotting a contour map is handled by :meth:`pygmt.Figure.grdcontour`
+"""
+
+import pygmt
+
+########################################################################################
+# Create contour plot
+# -------------------
+#
+# The grdcontour method takes the grid input and the region region values.
+# It plots annotated contour lines, which are darker and have the elevation/depth written on them,
+# and unannotated contour lines. In the example blow, the default contour line intervals are 500 meters,
+# with an annotated contour line every 1000 meters.
+# By default, it plots the map with a Mercator projection and with no frame.
```

Comment:
The default projection is `-JQ` (Equidistant cylindrical), not Mercator.

Reply:
@seisman My mistake. Do you need me to commit the change and submit a new PR?

Reply:
For some comments, you can just click the "commit suggestions" button:
![image](https://user-images.githubusercontent.com/3974108/100277445-28d6c980-2f31-11eb-83eb-11e8d7066b9e.png)

You can also push more changes directly to your "contour-map" branch. No need to submit a new PR.

Reply:
Thanks for the help; completed the commits directly on GitHub